src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -181,7 +181,7 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
         // Set selling functionality to open if true. By default selling is false
         sellIsOpen = bondingCurveProperties.sellIsOpen;
 
-        emit CollateralTokenSet(_acceptedToken, collateralTokenDecimals);
+        emit ReserveTokenSet(_acceptedToken, collateralTokenDecimals);
```

Comment:
Is this naming something we all agree on? I know that we call it collateral token everywhere else, so this looks a bit "off" to me.

Reply:
Yea, I was not sure about the naming. I just wanted to use the same event for both BondingCurve and Rebasing_FM when it comes to the token that both modules. My understanding is that the token that both modules take can be considered a reserve token, I wasn't so sure if the token can also be called a collateral token for the rebasing fm, but if so I'd also prefer to call it CollateralToken.